### PR TITLE
escapes wildcard - related #163

### PR DESCRIPTION
**Pull request by hbobenicio**
Simple fix related to a missing escape of a bash wildcard during docker images dowload checking.

### CODE REVIEW

**Comment by hbobenicio** [comment]
FIY @diestrin

**Comment by hbobenicio** [comment]
Should I have to sign-off my commit before merging?

**Comment by hbobenicio** [comment]
Am I missformatting the commit? Should it assign the commit with pgp or not? Tried some changes in the commit to satisfy DCO bot but didn't worked and I don't get why... :(

**Comment by diestrin** [comment]
Hey @hbobenicio, you need to signoff the commits, you can fix it with `git commit --amend --signoff --no-edit`

**Comment by hbobenicio** [comment]
@diestrin , done!

**Review by diestrin — APPROVED** [review]

**Comment by diestrin** [comment]
Thank you!

**Comment by diestrin** [comment]
I released your changes on #166

**Pull request merged**
